okhttp/src/main/java/okhttp3/internal/http2/Http2Connection.kt

```diff
@@ -433,7 +433,7 @@ class Http2Connection internal constructor(builder: Builder) : Closeable {
 
   /**
    * Closes this connection. This cancels all open streams and unanswered pings. It closes the
-   * underlying input and output streams and shuts down internal executor services.
+   * underlying input and output streams and shuts down internal executor services and task queues.
```

Comment:
Didn't your last commit remove "internal executor services"?

Reply:
Unfortunately there’s one more to sweep up that handles incoming pushes. That’s on the to-do list!